pkg/controller/operator.go

```diff
@@ -82,24 +84,45 @@ func (s *Operator) Run(ctx context.Context, controller *controllercmd.Controller
 		return err
 	}
 
+	desiredVersion := os.Getenv("RELEASE_VERSION")
+	missingVersion := "0.0.1-snapshot"
```

Comment:
This corresponds to https://github.com/openshift/insights-operator/blob/master/manifests/07-cluster-operator.yaml#L15, right? And it's updated by the CVO, right?